charts/aws-efs-csi-driver/templates/controller-serviceaccount.yaml

```diff
@@ -30,7 +30,7 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
     resources: ["events"]
-    verbs: ["list", "watch", "create"]
+    verbs: ["list", "watch", "create", "patch"]
```

Comment:
could you also run `make generate-kustomize` so the kustomize template also picks up this change? thx

Reply:
Done